.github/ISSUE_TEMPLATE/resource_donating.md

```diff
@@ -0,0 +1,32 @@
+Thank you for your donation!
+
+Leave your contact information below, our marketing will contact you later.
+
+### Full Name
+
+### Email
+
+### Organization
+
+### Job Title
+
+### Phone
+
+### Can logo be placed on openlabtesting.org?
+- [ ] Yes
+- [ ] No
+
+### If no, how to provide recognition?
+
+### Interested in sponsoring any of the following?
+- [ ] Additional Infrastructure
+- [ ] Engineering Support
+- [ ] Product/Project Manager(s)
+- [ ] QA, QE, and test builders
```

Comment:
- [ ] Engineering Support
- [ ] Product/Project Manager(s)
- [ ] QA, QE, and test builders

these look like a bit duplicated with the `joinin` section.